config_defaults/subtests/docker_cli/start.ini

```diff
@@ -8,6 +8,8 @@ docker_run_timeout = 30.0
 docker_attach = yes
 #: Run interactive container -i
 docker_interactive = yes
+#: docker command wait time
```

Comment:
@afomm  Yep, ed's right, but I wanted to get this fix included asap.

So, now, at your leasure, please open another PR that just fixes the comment (documentation), and we'll merge it in separately.  Thx!